doc/source/build-actions/index.rst

```diff
@@ -0,0 +1,49 @@
+Build actions
+=============
+
+The build actions allow for building artifacts for a Python library. These
+artifacts include both source distribution files and wheels.
+
+
+Build library action
+--------------------
+This action allows for building source and wheel artifacts for a Python library.
+
+.. jinja:: build-library
+
+    {{ inputs_table }}
+
+
+Code sample for using this action:
+
+.. code-block:: yaml
+
+    build-library:
+      name: Build library
+      runs-on: ubuntu-latest
+      steps:
+        - name: "Build library source and wheel artifacts"
+          uses: pyansys/actions/build-library@main
+          with:
+            library-name: "ansys-<product>-<library"
+
+
+Build C-extension library action
+--------------------------------
+This action allows for building wheel artifacts for a Python library using
+a C-extension.
+
+.. jinja:: build-ci-wheels
+
+    {{ inputs_table }}
+
+
+.. code-block:: yaml
+
+    build-c-extension:
+      name: Build a C-extension library
+      runs-on: ubuntu-latest
```

Comment:
```suggestion
      runs-on: ${{ matrix.os }}
      strategy:
         matrix:
             os: [ubuntu-latest, windows-latest, macos-11]
```
Just an example that the user can build wheels for any os. But adding the matrix in example or not is up to your call.